network-mux/CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # Revision history for mux
 
+## 0.2.0.0 -- 2023-01-25
```

Comment:
It seems `0.2.0.0` is already in `CHaP`: https://github.com/input-output-hk/cardano-haskell-packages/blob/main/_sources/network-mux/0.2.0.0/meta.toml.

We can skip the changelogs between `0.1` and `0.2`, but let's add what was changed since `0.2`.

Reply:
How? Is it not up-to-date? Should we bump the rev instead?

Reply:
Nevermind just re-read your message